### PR TITLE
Node.js quickstart example - changing keyword declaration type

### PR DESCRIPTION
**Pull request by pacostas**
Changing Keyword type declaration type of all used variables of the Node.js quickstart example, from var to const.

Closes keycloak/keycloak-quickstarts#288

### CODE REVIEW

**Review by abstractj — APPROVED** [review]

**Comment by abstractj** [comment]
@jonkoops LGTM, but I will wait for you approval prior to merge.

**Comment by jonkoops** [comment]
LGTM :heavy_check_mark:

**Pull request merged**
